app/components/documents/document-card.js

```diff
@@ -98,15 +97,20 @@ export default class DocumentsDocumentCardComponent extends Component {
       this.documentContainer = yield this.piece.documentContainer;
       this.accessLevel = yield this.piece.accessLevel;
     } else if (this.args.documentContainer) {
+      // TODO KAS-2777 This else does not seem used (no <Documents::DocumentCard> that passes this arg)
       this.documentContainer = this.args.documentContainer;
       yield this.loadVersionHistory.perform();
+      // TODO KAS-2777 does this work? Where is this.piece coming from if args.piece was not given?
       this.piece = yield loadPiece(this.piece.id);
       this.accessLevel = yield this.piece.accessLevel;
```

Comment:
@MikiDi Ik kwam dit tegen en ik snapte niet goed waarom dit nodig was.
Desnoods bekijken in een ander ticket en voorlopig afblijven, maar volgens mij zou dit niet werken as-is